logstash.go

```diff
@@ -39,41 +68,61 @@ func NewLogstashAdapter(route *router.Route) (router.LogAdapter, error) {
 
 // Stream implements the router.LogAdapter interface.
 func (a *LogstashAdapter) Stream(logstream chan *router.Message) {
+
+	strTags := getopt("LOGSTASH_TAGS", "")
+	tags := strToSlice(strTags, ",")
+
 	for m := range logstream {
+
 		dockerInfo := DockerInfo{
 			Name:     m.Container.Name,
 			ID:       m.Container.ID,
 			Image:    m.Container.Config.Image,
 			Hostname: m.Container.Config.Hostname,
 		}
+		debug("logstash.Stream(): dockerInfo = Name:", m.Container.Name)
+		debug("logstash.Stream(): dockerInfo = ID:", m.Container.ID)
+		debug("logstash.Stream(): dockerInfo = Image:", m.Container.Config.Image)
+		debug("logstash.Stream(): dockerInfo = Hostname:", m.Container.Config.Hostname)
 
 		var js []byte
 		var data map[string]interface{}
+
+		// Parse JSON-encoded m.Data
 		if err := json.Unmarshal([]byte(m.Data), &data); err != nil {
 			// The message is not in JSON, make a new JSON message.
 			msg := LogstashMessage{
 				Message: m.Data,
 				Docker:  dockerInfo,
 				Stream:  m.Source,
+				Tags:    tags,
 			}
+			debug("logstash.Stream(): msg = m.Data:", m.Data)
+			debug("logstash.Stream(): msg = dockerInfo:", dockerInfo)
+			debug("logstash.Stream(): msg = m.Source:", m.Source)
+			debug("logstash.Stream(): msg = tags:", tags)
+
 			if js, err = json.Marshal(msg); err != nil {
-				log.Println("logstash:", err)
-				continue
```

Comment:
hey @maxekman I should have created a different commit for that, anyways the ```continue``` will enforce to start the next loop before writing to logstash, what doesn't make much sense.
You wanna make sure to write to logstash before parsing next line from ```logstream```.